### PR TITLE
Fixed broken links - 2024.07

### PR DESCRIPTION
**Pull request by n-boshnakov**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
This PR:
- fixes broken links
- pretties up the markdown

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

File: docs/tutorials/tutorial-custom-domain-with-istio.md [hunk 1]
```diff
@@ -221,16 +232,19 @@ I want to install Istio with a default profile and SDS enabled. Furthermore I pa
...
+
 1. The [External DNS Manager](https://github.com/gardener/external-dns-management/blob/master/README.md), provided to you as a service (`dns.gardener.cloud/class: garden`), picks up the request and creates the wildcard DNS entry `*.gsicdc.mydomain.io` with a time to live of 120sec at your DNS provider. My provider Cloud Flare is very very quick (as opposed to some other services). You should be able to verify the entry with `dig lovemygardener.gsicdc.mydomain.io` within seconds.
-2. The [Certificate Management](https://github.com/gardener/cert-management/blob/master/README.md) picks up the request as well and initates a DNS01 protocol exchange with Let's Encrypt; using the staging environment referred to with the issuer behind `mydomain-staging`.
-3. After aproximately 70sec (give and take) you will receive the wildcard certificate in the `wildcard-tls` secret in the namespace `istio-system`.
+1. The [Certificate Management](https://github.com/gardener/cert-management/blob/master/README.md) picks up the request as well and initiates a DNS01 protocol exchange with Let's Encrypt; using the staging environment referred to with the issuer behind `mydomain-staging`.
```
Comment (MartinWeindel) [inline docs/tutorials/tutorial-custom-domain-with-istio.md:239]: Fix numbering
```suggestion
2. The [Certificate Management](https://github.com/gardener/cert-management/blob/master/README.md) picks up the request as well and initiates a DNS01 protocol exchange with Let's Encrypt; using the staging environment referred to with the issuer behind `mydomain-staging`.
```

File: docs/tutorials/tutorial-custom-domain-with-istio.md [hunk 2]
```diff
@@ -221,16 +232,19 @@ I want to install Istio with a default profile and SDS enabled. Furthermore I pa
...
+1. After aproximately 70sec (give and take) you will receive the wildcard certificate in the `wildcard-tls` secret in the namespace `istio-system`.
```
Comment (MartinWeindel) [inline docs/tutorials/tutorial-custom-domain-with-istio.md:240]: Fix numbering
```suggestion
3. After aproximately 70sec (give and take) you will receive the wildcard certificate in the `wildcard-tls` secret in the namespace `istio-system`.
```

File: docs/tutorials/tutorial-custom-domain-with-istio.md [hunk 3]
```diff
@@ -347,7 +363,7 @@ spec:
         ports:
         - containerPort: 80
 ---
-apiVersion: networking.istio.io/v1
+apiVersion: networking.istio.io/v1beta1
```
Comment (MartinWeindel) [inline docs/tutorials/tutorial-custom-domain-with-istio.md:366]: Why are you not staying with the current version here?

File: docs/tutorials/tutorial-custom-domain-with-istio.md [hunk 4]
```diff
@@ -374,7 +390,7 @@ spec:
     hosts:
     - "httpbin.gsicdc.mydomain.io"
 ---
-apiVersion: networking.istio.io/v1
+apiVersion: networking.istio.io/v1beta1
```
Comment (MartinWeindel) [inline docs/tutorials/tutorial-custom-domain-with-istio.md:393]: Why v1beta1?

File: docs/tutorials/tutorial-custom-domain-with-istio.md [hunk 5]
```diff
@@ -292,13 +308,13 @@ brew install derailed/k9s/k9s
...
 {{% /alert %}}
 
-Kubernetes Ingress is a subject that is evolving to much broader standard. Please watch [Evolving the Kubernetes Ingress APIs to GA and Beyond](https://www.youtube.com/watch?v=cduG0FrjdJA) for a good introduction. In this example, I did not want to use the Kubernetes `Ingress` compatibility option of Istio. Instead, I used `VirtualService` and `Gateway` from the Istio's API group `networking.istio.io/v1` directly, and enabled istio-injection generically for the namespace.
+Kubernetes Ingress is a subject that is evolving to much broader standard. Please watch [Evolving the Kubernetes Ingress APIs to GA and Beyond](https://www.youtube.com/watch?v=cduG0FrjdJA) for a good introduction. In this example, I did not want to use the Kubernetes `Ingress` compatibility option of Istio. Instead, I used `VirtualService` and `Gateway` from the Istio's API group `networking.istio.io/v1beta1` directly, and enabled istio-injection generically for the namespace.
```
Comment (MartinWeindel) [inline docs/tutorials/tutorial-custom-domain-with-istio.md:317]: Why are you not staying with the current version `v1` here?

**Review by MartinWeindel — CHANGES_REQUESTED** [review]
Thanks for the improving the tutorial.
I have two nits:
- Please fix the numbering which has gone mixed up
- Can you explain, why you changed to version `v1beta1` instead of keeping the current version `networking.istio.io/v1`?

File: docs/tutorials/tutorial-custom-domain-with-istio.md (hunk 1 above)
Comment (n-boshnakov) [reply docs/tutorials/tutorial-custom-domain-with-istio.md:239]: Numbering an ordered list as 

```
1. Item
1. Item
1. Item
```

will show up as:
1. Item
1. Item
1. Item

That way you do not have to keep track of which number each item is, and can focus on the actual content.

File: docs/tutorials/tutorial-custom-domain-with-istio.md (hunk 3 above)
Comment (n-boshnakov) [reply docs/tutorials/tutorial-custom-domain-with-istio.md:366]: I do not know why the version got changed, maybe it's an issue of documentation versioning. I will fix it.

File: docs/tutorials/tutorial-custom-domain-with-istio.md (hunk 1 above)
Comment (MartinWeindel) [reply docs/tutorials/tutorial-custom-domain-with-istio.md:239]: Good to know!

File: docs/tutorials/tutorial-custom-domain-with-istio.md [hunk 6]
```diff
@@ -97,21 +105,22 @@ The next snipplet allows Gardener to manage certificates automatically from *[Le
...
 * [Wildcard Certificates](https://community.letsencrypt.org/t/acme-v2-production-environment-wildcards/55578)
 
 ## Create the Gardener Shoot Cluster
 
-Remember I chose to create the Shoot on GCP, so below is the simplest declarative shoot or cluster order document. Notice that I am referring to the infrastructure credentials with `shoot-operator-gcp` and I combined the above snipplets into the yaml file:
+Remember I chose to create the Shoot on GCP, so below is the simplest declarative shoot or cluster order document. Notice that I am referring to the infrastructure credentials with `shoot-operator-gcp` and I combined the above snippets into the yaml file:
 
 ```yaml
-apiVersion: core.gardener.cloud/v1beta1
+apiVersion: core.gardener.cloud/v1
```
Comment (n-boshnakov) [inline docs/tutorials/tutorial-custom-domain-with-istio.md:123]: Do we want to use `v1beta1` here of `v1`?

Comment (MartinWeindel) [reply docs/tutorials/tutorial-custom-domain-with-istio.md:123]: For Shoots we still have API Version `core.gardener.cloud/v1beta1`, that is correct as it is.

**Review by MartinWeindel — DISMISSED** [review]
/lgtm

Thanks for your contribution!

**Comment by n-boshnakov** [comment]
I reverted the version of the shoots to `v1beta1`

**Review by MartinWeindel — APPROVED** [review]

**Pull request merged**